libtelemetry/src/test/java/com/mapbox/android/telemetry/TelemetryClientTest.java

```diff
@@ -257,6 +257,20 @@ public void checksDebugLoggingEnabledAttachment() throws Exception {
       .debug(eq("TelemetryClient"), contains(" with 1 event(s) (user agent: anyUserAgent) with payload:"));
   }
 
+  @Test
```

Comment:
I can add a regex to check if entered url is valid. Will it be ok?

Reply:
@andrlee can we remove 
`private static final String HTTPS_SCHEME = "https";`
and add `https` to every default host? 
so in a method setBaseUrl() we will pass a url with `https` prefix and will check if url is valid with regex

because if we pass a string without `https` (and add it later), it is hard to check if the string is valid url

does it make sense?

Reply:
or it is required to set HttpUrl scheme in a separate method?